app/helpers/task_list_helper.rb

```diff
@@ -18,7 +18,8 @@ def task_list_includes?(legal_aid_application, task_name)
 private
 
   def _task_url(name, legal_aid_application, status)
-    url = if application_has_no_involved_children?(legal_aid_application) && name.eql?(:children_application)
+    url = if (application_has_no_involved_children?(legal_aid_application) && name.eql?(:children_application)) ||
+        (application_has_no_opponents?(legal_aid_application) && name.eql?(:opponent_name))
```

Comment:
Complexity: I know our domain is complicated and this feels complex, is it be possible to make this less complex?

Reply:
Have a look at the latest version @naseberry - happy to discuss